Libraries/LibWeb/IndexedDB/Internal/ObjectStore.cpp

```diff
@@ -89,4 +89,18 @@ void ObjectStore::clear_records()
     m_records.clear();
 }
 
+AK::Vector<Record> ObjectStore::first_n_in_range(GC::Ref<IDBKeyRange> range, size_t count)
```

Comment:
It is not safe to be returning a Vector of a struct which contains GC objects. This should either be a ConservativeVector, or GC::Root the record type.